unstructured_inference/inference/layout.py

```diff
@@ -76,7 +77,13 @@ def from_file(cls, filename: str, model: Optional[Detectron2LayoutModel] = None)
     def from_image_file(cls, filename: str, model: Optional[Detectron2LayoutModel] = None):
         """Creates a DocumentLayout from an image file."""
         logger.info(f"Reading image file: {filename} ...")
-        image = Image.open(filename)
+        try:
+            image = Image.open(filename)
+        except Exception as e:
+            if os.path.isdir(filename) or os.path.isfile(filename):
```

Comment:
If it's a directory, should we tell them it's a directory?

Reply:
From my testing passing a directory in this context gets handled properly by `Image.open`, raising an `IsADirectoryError`. I'll add a test case for this though.